fabric-lifecycle-events-v1/src/client/java/net/fabricmc/fabric/api/client/event/lifecycle/v1/CameraUpdateCallback.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.event.lifecycle.v1;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.world.BlockView;
+import net.minecraft.client.render.Camera;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+/**
+ * Callback for after the game camera is updated.
```

Comment:
The javadoc should describe what the event can do. I assume the goal is to modify the camera to implement various rendering effects?

Reply:
Copied from my response on discord:

> ah, right, sorry, meant to respond yesterday and forgot. Yeah, so the event is for modifying the camera after it's been updated for third person and for the sleep angle and position if the entity is sleeping. vanilla calls this in WorldRenderer#render to reset or setup the camera, so yes, it's reset to vanilla's handling if a mod stops modifying it